client/src/components/userMenu/userMenu.jsx

```diff
@@ -96,8 +95,8 @@ export class UserMenu extends React.Component {
 		if (this.props.user) {
 			const buttons = [];
 			
-			buttons.push(<img className="floatingLogo" src={addButton} alt="Add Sighting" onClick={this.props.addSighting}/>);
-			buttons.push(<img className="floatingLogo" src={historyButton} alt="Add Sighting" onClick={this.props.viewAnimals}/>);
+			buttons.push();
+			buttons.push();
 		
 			return buttons
```

Comment:
This function was from when floating buttons were at the top right, and the menu was going to go over them. I had it so that the buttons would disappear when the menu was opened, and re-appear when closed. I think we're probably safe to remove this now with buttons on the bottom.